src/components/ProjectSettingsTabs/AliasRow.tsx

```diff
@@ -71,90 +67,119 @@ export const AliasRow: SpruceFormProps["ObjectFieldTemplate"] = ({
+  const contents = (
+    <>
+      {alias.content}
+      <TaskRegexContainer>
+        <StyledSegmentedControl
+          label="Variant"
+          id="variant-input-control"
+          value={variantInput}
+          onChange={(value) => {
+            setVariantInput(value);
+            clearForm(value);
+          }}
+        >
+          <SegmentedControlOption
+            value={VariantInput.Tags}
+            disabled={isDisabled}
+            aria-controls={makeId(VariantInput.Tags)}
+          >
+            Tags
+          </SegmentedControlOption>
+          <SegmentedControlOption
+            value={VariantInput.Regex}
+            disabled={isDisabled}
+            aria-controls={makeId(VariantInput.Regex)}
+          >
+            Regex
+          </SegmentedControlOption>
+        </StyledSegmentedControl>
+        {variantInput === VariantInput.Tags ? (
+          <div data-cy="variant-tags-field" id={makeId(VariantInput.Tags)}>
+            {variantTags.content}
+          </div>
+        ) : (
+          <div data-cy="variant-field" id={makeId(VariantInput.Regex)}>
+            {variant.content}
+          </div>
+        )}
+      </TaskRegexContainer>
+      <TaskRegexContainer>
+        <StyledSegmentedControl
+          label="Task"
+          id="task-input-control"
+          value={taskInput}
+          onChange={(value) => {
+            setTaskInput(value);
+            clearForm(value);
+          }}
+        >
+          <SegmentedControlOption
+            value={TaskInput.Tags}
+            disabled={isDisabled}
+            aria-controls={makeId(TaskInput.Tags)}
+          >
+            Tags
+          </SegmentedControlOption>
+          <SegmentedControlOption
+            value={TaskInput.Regex}
+            disabled={isDisabled}
+            aria-controls={makeId(TaskInput.Regex)}
+          >
+            Regex
+          </SegmentedControlOption>
+        </StyledSegmentedControl>
+        {taskInput === TaskInput.Tags ? (
+          <div data-cy="task-tags-field" id={makeId(TaskInput.Tags)}>
+            {taskTags.content}
+          </div>
+        ) : (
+          <div data-cy="task-field" id={makeId(TaskInput.Regex)}>
+            {task.content}
+          </div>
+        )}
+      </TaskRegexContainer>
+    </>
+  );
+
+  return useExpandableCard ? (
+    <ExpandableCard
+      defaultOpen={!isDisabled}
+      contentClassName="patch-alias-card-content"
+      title={
+        <>
+          <TitleWrapper data-cy="expandable-card-title">
+            {initialAlias || accordionTitle}
+          </TitleWrapper>
+          {!readonly && (
+            <Button
+              onClick={onDropIndexClick(index)}
+              disabled={disabled || readonly}
+              leftGlyph={<Icon glyph="Trash" />}
+              data-cy="delete-item-button"
+              size="small"
+            />
+          )}
+        </>
+      }
+    >
+      {contents}
+    </ExpandableCard>
+  ) : (
     <Accordion
```

Comment:
The alignment between the text and Delete button is a little off because of the margins on the `AccordionTitle`. 

<img width="418" alt="Screen Shot 2022-02-14 at 1 23 12 PM" src="https://user-images.githubusercontent.com/47064971/153923725-0e978487-8250-4521-bf70-f2c436a3e2da.png">

One way to fix it is by changing the `align-items` property on the `ArrayItemRow` to `baseline` but I'm not sure that's what you want in this situation... 

<img width="304" alt="Screen Shot 2022-02-14 at 1 19 58 PM" src="https://user-images.githubusercontent.com/47064971/153924589-f84b8f16-cbb6-431e-bb15-9a03adac9b86.png">

Reply:
That had `align-items` rule had some unexpected ramifications, so I opted here to use a custom margin around the title to force alignment. Let me know what you think!

Reply:
Oh yeah hmm I see the problem. Is there any reason this Delete button can't be with the `Accordion` title like how it's with the `ExpandableCard` title in `AliasRow.tsx`? From a glance the only difference I can see is the usage of `topAlignDelete`, which I wonder if that can be propagated down since `AliasRow` also has access to `uiSchema`.